### PR TITLE
Fix Makefile error handling

### LINKED ISSUE #734: make uninstall fails if /etc/opencas/opencas.conf is removed
## Description
Uninstallation is interrupted if opencas.conf is removed beforehand. In effect not all files are removed.

## Expected Behavior
Finished uninstallation

## Actual Behavior
Error No such file or directory

## Steps to Reproduce
1. Remove /etc/opencas/opencas.conf
2. Uninstall Open-CAS

## Context
I'm trying to achieve clean uninstallation

## Workaround
make install && make uninstall

## Logs
```[root@localhost open-cas-linux]# make uninstall
cd modules && make uninstall
make[1]: Entering directory '/root/open-cas-linux/modules'
Uninstalling Open-CAS modules
make[1]: Leaving directory '/root/open-cas-linux/modules'
cd casadm && make uninstall
make[1]: Entering directory '/root/open-cas-linux/casadm'
Uninstalling casadm
rm: cannot remove '/etc/opencas/opencas.conf': No such file or directory
make[1]: *** [Makefile:153: uninstall] Error 1
make[1]: Leaving directory '/root/open-cas-linux/casadm'
make: *** [Makefile:22: casadm] Error 2
```

## Your Environment
* OpenCAS version (commit hash or tag): any
* Operating System: any
* Kernel version: any

### PR DESCRIPTION
**Pull request by robertbaldyga**
Fixes #734

### CODE REVIEW

File: modules/Makefile [hunk 1]
```diff
@@ -15,6 +15,13 @@ obj-y += cas_disk/
 # line; invoke the kernel build system.
 else
 
+include ../tools/helpers.mk
+
+define remove-module
+	@if (lsmod | grep -q ${1}); then rmmod ${1}; \
+	else echo "WARNING: Module ${1} is not loaded"; fi
+endef
```
Comment (rafalste) [inline modules/Makefile:23]: Why not move it to the _helpers.mk_ like the other defines for _remove-file_ and _remove-directory_?

**Review by rafalste — APPROVED** [review]

File: modules/Makefile (hunk 1 above)
Comment (robertbaldyga) [reply modules/Makefile:23]: This function isn't used in any other Makefile, so there is no need to move it to common code.

File: utils/Makefile [hunk 2]
```diff
@@ -30,6 +32,14 @@ else
 install_files:
 	@echo "Installing Open-CAS utils"
 
+	@install -m 755 -d $(DESTDIR)/etc/opencas
+	@install -m 644 $(UTILS_DIR)/opencas.conf $(DESTDIR)/etc/opencas/opencas.conf
+	@install -m 444 $(UTILS_DIR)/ioclass-config.csv $(DESTDIR)/etc/opencas/ioclass-config.csv
```
Comment (mmichal10) [inline utils/Makefile:39]: I know it is just copy-paste, but is there any particular reason why `ioclass-config.csv` is installed with permissions different than the rest of config files? How about changing it to `644`?

File: Makefile [hunk 3]
```diff
@@ -27,16 +27,16 @@ endif
 endif
 
 archives:
-	@utils/pckgen $(PWD) tar zip
```
Comment (mmichal10) [inline Makefile:30]: I think naming the directory `build_tools` would make things more clear.

**Review by mmichal10 — APPROVED** [review]

**Pull request merged**